src/txn/txn_rollback_to_stable.c

```diff
@@ -355,14 +355,28 @@ __rollback_abort_row_ondisk_kv(
     WT_DECL_RET;
     WT_ITEM buf;
     WT_UPDATE *upd;
-    char ts_string[4][WT_TS_INT_STRING_SIZE];
+    char ts_string[5][WT_TS_INT_STRING_SIZE];
     bool prepared;
 
     vpack = &_vpack;
     upd = NULL;
     __wt_row_leaf_value_cell(session, page, rip, NULL, vpack);
     prepared = vpack->tw.prepare;
-    if (vpack->tw.durable_start_ts > rollback_timestamp ||
+    if (WT_IS_HS(S2BT(session))) {
+        if (vpack->tw.durable_stop_ts > rollback_timestamp) {
+            __wt_verbose(session, WT_VERB_RTS,
+              "hs update aborted with start durable timestamp: %s, commit timestamp: %s, "
```

Comment:
Instead of adding start/stop to every timestamp, I changed something like the following, how do you think?
`hs update aborted with start durable/commit timestamp: %s, %s`